config/icons.js

```diff
@@ -12,15 +12,17 @@ module.exports = function() {
     'free-brands-svg-icons': [
       'twitter'
     ],
-    'pro-solid-svg-icons': [
+    'free-solid-svg-icons': [
       'chevron-left',
       'cog',
       'sort',
       'sort-down',
       'sort-up',
-      'badge-check',
+      'asterisk',
+      'certificate',
       'exclamation-triangle',
+      'calendar-plus',
```

Comment:
I assume these two icons were the other options you considered? Can you please remove them from the file to avoid pulling them into the compiled app please :)

Reply:
😅 Whoops, yes!